packages/@react-aria/utils/src/mergeRefs.ts

```diff
@@ -10,12 +10,12 @@
  * governing permissions and limitations under the License.
  */
 
-import {ForwardedRef} from 'react';
+import {ForwardedRef, MutableRefObject} from 'react';
 
 /**
  * Merges multiple refs into one. Works with either callback or object refs.
  */
-export function mergeRefs<T>(...refs: ForwardedRef<T>[]): ForwardedRef<T> {
+export function mergeRefs<T>(...refs: Array<ForwardedRef<T> | MutableRefObject<T>>): ForwardedRef<T> {
```

Comment:
had to update the typing for this but not super sure about this change

Reply:
react-aria/utils is in TS strict, so we're pretty well protected here
that said, I think it's the right change